daemon/events/metrics.go

```diff
@@ -0,0 +1,21 @@
+package events
```

Comment:
Creating a separate file for metrics is a smell, metrics should be treated like logging and live in the same file as what they're instrumenting.

Similarly that the same metric is being updated in multiple files is a smell, indicating that you're not instrumenting at the right place in your code. Either you shouldn't be using a label, or you should move the instrumentation up the call stack so each metric only lives in one file.